docs/features/nullable-reference-types.md

```diff
@@ -40,9 +40,18 @@ Unannotated reference types are non-nullable or null-oblivious depending on whet
 ```c#
 namespace System.Runtime.CompilerServices
 {
-    [AttributeUsage(
-        AttributeTargets.All,
-        AllowMultiple = false)]
+    [AttributeUsage(AttributeTargets.Class |
```

Comment:
Why not assembly?

Reply:
From our discussion, I added a [note](https://github.com/dotnet/roslyn/issues/22152#issuecomment-334331966) to work items (`module:` target is indeed unfamiliar to developers).